SerializedVersion.java

```diff
@@ -0,0 +1,93 @@
+import java.util.*; 
+import java.lang.*; 
+import java.io.*; 
+  
+class SerializedVersion { 
+    // A utility function to find the vertex with minimum distance value, 
+    // from the set of vertices not yet included in shortest path tree 
+    static final int V = 9; 
+    int minDistance(int dist[], Boolean sptSet[]) 
+    { 
+        // Initialize min value 
+        int min = Integer.MAX_VALUE, min_index = -1; 
+  
+        for (int v = 0; v < V; v++) 
+            if (sptSet[v] == false && dist[v] <= min) { 
+                min = dist[v]; 
+                min_index = v; 
+            } 
+  
+        return min_index; 
+    } 
+  
+    // A utility function to print the constructed distance array 
+    void printSolution(int dist[]) 
+    { 
+        System.out.println("Vertex \t\t Distance from Source"); 
+        for (int i = 0; i < V; i++) 
+            System.out.println(i + " \t\t " + dist[i]); 
+    } 
+  
+    // Function that implements Dijkstra's single source shortest path 
+    // algorithm for a graph represented using adjacency matrix 
+    // representation 
+    void dijkstra(int graph[][], int src) 
```

Comment:
Pls change the signature of this method to be same as the parallelized one

Reply:
Done